executors.go

```diff
@@ -58,6 +94,22 @@ var emptyUUID = uuid.Nil.String()
 // Save wraps the Create and Update methods. It executes a Create if no ID is provided with the entry;
 // or issues an Update otherwise.
 func (c *Connection) Save(model interface{}, excludeColumns ...string) error {
+	v := reflect.Indirect(reflect.ValueOf(model))
```

Comment:
I feel like a lot of this logic is repeated. would make sense to break it out into it's own method?